src/index.html

```diff
@@ -52,30 +52,33 @@
 
     <!-- Footer -->
     <footer>
-      <div>
+      <div class="logos">
         <!-- Logos can be added here -->
         <p style="margin-bottom: 4px">
           <img
-            src="./assets/content/images/angular-bros.png"
+            ngSrc="./assets/content/images/angular-bros.png"
             alt="Angular Bros"
+            loading="lazy"
             style="margin-right: 15px; opacity: 0.8"
             width="132"
             height="44"
           />
         </p>
         <p style="margin-bottom: 4px">
           <img
-            src="https://www.drupal.org/files/styles/grid-4-2x/public/Packt%20Logo1_0.png?itok=p5jrImHT"
+            ngSrc="https://www.drupal.org/files/styles/grid-4-2x/public/Packt%20Logo1_0.png?itok=p5jrImHT"
             alt="Angular Space"
+            loading="lazy"
             style="margin-right: 15px; opacity: 0.8"
             width="132"
             height="44"
           />
         </p>
         <p style="margin-bottom: 4px">
           <img
-            src="./assets/content/images/angular-space-footer.png"
+            ngSrc="./assets/content/images/angular-space-footer.png"
```

Comment:
<img width="657" alt="image" src="https://github.com/user-attachments/assets/d5f56806-4ab2-4691-ae3c-7124fcf5ac40">
🤔

Reply:
images not loaded on netlify?